extensions/wikia/Search/classes/IndexService/CrossWikiCore.php

```diff
@@ -65,113 +69,130 @@ protected function getWikiBasics() {
-			$result = [ 'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'], 'views_monthly_i' => $wvResponse['contents']['wikiviews_monthly']['set'] ];
+		if ( $wvResponse = ( new WikiViews )->getStubbedWikiResponse() ) {
+			$result = [
+				'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'],
+				'views_monthly_i' => $wvResponse['contents']['wikiviews_monthly']['set']
+			];
 		}
+
 		return $result;
 	}
-	
+
 	/**
 	 * Retrieves WAM score for this wiki
+	 *
 	 * @return array
 	 */
 	protected function getWam() {
 		$response = [];
-		if ( $wamResp = (new Wam)->getStubbedWikiResponse() ) {
+		if ( $wamResp = ( new Wam )->getStubbedWikiResponse() ) {
 			$response['wam_i'] = $wamResp['contents']['wam']['set'];
 		}
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves stats like number of videos, number of images, number of articles, etc.
+	 *
 	 * @return array
 	 */
 	protected function getWikiStats() {
 		$service = $this->getService();
 		$data = $service->getApiStatsForWiki();
 		$response = [];
-		if ( (! empty( $data['query'] ) ) && ( isset( $data['query']['statistics'] ) ) ) {
+		if ( ( !empty( $data['query'] ) ) && ( isset( $data['query']['statistics'] ) ) ) {
 			foreach ( $data['query']['statistics'] as $key => $val ) {
 				$response[$key . '_i'] = $val;
 			}
 		}
-		$response['videos_i'] = (new WikiService)->getTotalVideos( $this->getWikiId() );
+		$response['videos_i'] = ( new WikiService )->getTotalVideos( $this->getWikiId() );
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves description, headline, etc.
+	 *
 	 * @return array
 	 */
 	protected function getVisualizationInfo() {
 		$response = [];
 		$service = $this->getService();
 
-		$message = $service->getSimpleMessage( 'wikiasearch2-crosswiki-description', array( $service->getGlobal( 'Sitename' ) ) );
+		$message =
+			$service->getSimpleMessage( 'wikiasearch2-crosswiki-description', [ $service->getGlobal( 'Sitename' ) ] );
 		$ds = Utilities::field( 'description' );
 		$ds = $ds == 'description' ? 'description_txt' : $ds;
 		$response[$ds] = $message;
 		$response['description_txt'] = $message;
 		$vizInfo = $service->getVisualizationInfoForWikiId( $this->getWikiId() );
-		if (! empty( $vizInfo ) ) {
+		if ( !empty( $vizInfo ) ) {
 			$response['image_s'] = $vizInfo['image'];
 			if ( isset( $vizInfo['desc'] ) ) {
 				$response[$ds] = $vizInfo['desc'];
 				$response['description_txt'] = $vizInfo['desc'];
 			}
 			foreach ( $vizInfo['flags'] as $flag => $bool ) {
-				$response[$flag.'_b'] = $bool ? 'true' : 'false';
+				$response[$flag . '_b'] = $bool ? 'true' : 'false';
 			}
 			$response['headline_txt'] = $vizInfo['headline'];
 		}
+
 		return $response;
 	}
-	
+
 	/**
 	 * Accesses the ArticlesApiController to store the top articles for this wiki
+	 *
 	 * @return array
 	 */
 	protected function getTopArticles() {
-		$response = ['top_articles_txt' => []];
+		$response = [ 'top_articles_txt' => [] ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "top_articles_txt" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)